accounts/addproofer.php

```diff
@@ -105,7 +107,7 @@
 // to this file & run the above commands.
 
 $header = _("Create An Account");
-output_header($header);
+output_header($header, true, array("js_files" => array("$code_url/accounts/addproofer.js")));
```

Comment:
Please use the constant  `SHOW_STATSBAR` (which is just `TRUE`) for the second argument as it's a little clearer on what that is.